pontoon/sync/changeset.py

```diff
@@ -397,6 +410,12 @@ def bulk_update_entities(self):
 
     def bulk_create_translations(self):
         Translation.objects.bulk_create(self.translations_to_create)
+        for translation in self.translations_to_create:
+            log_action(
+                'translation:created',
+                translation.user or self.sync_user,
+                translation=translation,
+            )
```

Comment:
We must use `bulk_create` here, same as we do for TM and checks.

Otherwise adding a locale to Firefox or running a Fluent migration will result in tens of thousands of new DB calls.